An up-to-date Startup.pdf is shipped with the dist, so just use that.
Patch taken from Void Linux:
https://github.com/void-linux/void-packages/blob/master/srcpkgs/apvlv/patches/use_dist_manpdf.patch

diff -rup CMakeLists.txt.orig CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,10 +76,6 @@ IF (WIN32)
     INSTALL(DIRECTORY icons DESTINATION ".")
     INSTALL(DIRECTORY ${CMAKE_HOME_DIRECTORY}/win32/poppler-data DESTINATION ".")
 ELSE (WIN32)
-    ADD_CUSTOM_TARGET(Startup.pdf
-            ALL
-            COMMAND "man" "-t" "${CMAKE_SOURCE_DIR}/apvlv.1" "|" "ps2pdf" "-" "${CMAKE_SOURCE_DIR}/Startup.pdf"
-            DEPENDS apvlv.1)
     INSTALL(FILES apvlvrc.example DESTINATION ${SYSCONFDIR}
             RENAME apvlvrc)
     INSTALL(FILES apvlvrc.example Startup.pdf main_menubar.glade DESTINATION ${DOCDIR})
